core/src/main/java/org/bitcoinj/core/Locktime.java

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright by the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bitcoinj.core;
```

Comment:
I thought about it, and I certainly had our new rules in mind when wring this class. Currently it's closely tied to Transaction so I thought maybe move together with Transaction?

Reply:
I'll leave this choice up to you. If you think it's a good idea to move it to `base` right now (with `Transaction` likely not following this release), I'll move it.

Reply:
I agree with your instinct, let's wait until we move `Transaction`. It's not much use without it and it's possible there may be more changes when we change transaction. We're also not yet sure which package or module we will put `Transaction` in.  (e.g. it might require `crypto`.)